### PR TITLE
Remove the "stage" term in filters

### PR DESCRIPTION
**Pull request by McGiverGim**
Remove the "stage" term as discussed https://github.com/betaflight/blackbox-log-viewer/pull/173

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1034,23 +1034,23 @@ <h5 class="modal-title-craft"></h5>
                                         <tbody>
                                             <tr>
                                                 <td name='gyro_soft_type'>
-                                                    <label>Type Stage 1<br>&nbsp;</label>
+                                                    <label>Type Filter 1<br>&nbsp;</label>
```
Comment (mikeller) [inline index.html]: I'd probably go for 'Lowpass Filter 1 Type'. (and the same for 2)

File: index.html [hunk 2]
```diff
@@ -1034,23 +1034,23 @@ <h5 class="modal-title-craft"></h5>
...
                                                     <select>
                                                         <!-- list generated here -->
                                                     </select>
                                                 </td>
                                                 <td name="gyro_lowpass_hz">
-                                                    <label>Soft Stage 1<br>(Hz)</label>
+                                                    <label>Soft Filter 1<br>(Hz)</label>
```
Comment (mikeller) [inline index.html]: I'd probably go for 'Lowpass Filter 1 Cutoff'. (and the same for 2)

File: index.html [hunk 3]
```diff
@@ -1087,19 +1087,19 @@ <h5 class="modal-title-craft"></h5>
                                         <tbody>
                                             <tr>
                                                 <td name='dterm_filter_type'>
-                                                    <label>Type Stage 1
+                                                    <label>Type Filter 1
```
Comment (mikeller) [inline index.html]: 'Lowpass Filter 1 Type'

File: index.html [hunk 4]
```diff
@@ -1087,19 +1087,19 @@ <h5 class="modal-title-craft"></h5>
...
                                                         <br>&nbsp;</label>
                                                     <select>
                                                         <!-- list generated here -->
                                                     </select>
                                                 </td>
                                                 <td name="dterm_lpf_hz">
-                                                    <label>Soft Stage 1
+                                                    <label>Soft Filter 1
```
Comment (mikeller) [inline index.html]: 'Lowpass Filter 1 Cutoff'

**Comment by McGiverGim** [comment]
The only problem is that I need to divide it in two lines. The "auto-adjust" of the header page is very bad.
So it will be like that:
>Lowpass Filter 1
>Type

Or something like this. Is right in this way?

**Comment by mikeller** [comment]
Definitely, looks good.

**Comment by McGiverGim** [comment]
Needed to add the number in the second line, it was too long :(

Final result:
image

**Comment by McGiverGim** [comment]
I think I will move the cutoff of the notch to the second line too, to be all the same...

**Comment by McGiverGim** [comment]
Done:
image

**Pull request merged**